### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by noah-puckett**

### CODE REVIEW

File: html/index.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    <header>
+        <h1>You wake up,</h1>
+    </header>
+
+    <main>
+        <form id="form">
+            <p>
+                <label for="name">Who are you?</label>
+                    <input id="name" name="name" type="text">
+
```
Comment (Zilula) [inline html/index.html]: unneeded white space

File: html/index.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                    <input required type="radio" name="goal" id="work" value="work">
+                    go to work
+                    <img src="../media/spoon.png" alt="a picture of a spoon">
+                    <img src="../media/spoon.png" alt="a picture of a spoon">
+                    <img src="../media/spoon.png" alt="a picture of a spoon">
+                    <img src="../media/spoon.png" alt="a picture of a spoon">
+                    <img src="../media/spoon.png" alt="a picture of a spoon">
+                    </label>
+                </p>
+
```
Comment (Zilula) [inline html/index.html]: unneeded white space

Comment (noah-puckett) [reply html/index.html]: fixed! thank you!

File: html/index.html (hunk 1 above)
Comment (noah-puckett) [reply html/index.html]: gotcha! thanks!

File: map.html [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+
+
+
+
+
+
+
+
+<!-- <main>
```
Comment (Zilula) [inline map.html]: Don't leave commented out code in projects. If you want to save it, I'd suggest moving it another project, that you can use as a storage place.

File: quest.html [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+            </p>
+        </section>
+    </main>
+
+    <script type="module" src="./src/services/quest-data.js"></script>
+</body>
+</html>
+
+
+<!-- <label for="dress">
```
Comment (Zilula) [inline quest.html]: Same as the above comment. If it isn't being used, remove it so it doesn't cause confusion when you go back to it later.

**Pull request merged**